server/handlers/getSurvey.js

```diff
@@ -1,13 +1,15 @@
+
+/* eslint-ignore no-console */
 const readSurvey = require('../queries/readSurvey');
 
 const getSurvey = async (req, res) => {
   try {
     // checks if params contains key id or if params itself is just one string
-    const id = !req.params.id ? req.params : req.params.id;
-
+    const { id } = req.params
```

Comment:
i know this looks weird but im pretty sure it was the only way of fixing @Joaoviana bug with sending params. I would ask him before removing this

Reply:
if we're writing hacky stuff like this then it points to a deeper problem with our code, I'd rather we left it like this and if we encounter problems then we can address them

Reply:
cool, fine with that!